Jellyfin.Api/Controllers/DevicesController.cs

```diff
@@ -111,28 +115,31 @@ public async Task<ActionResult> UpdateDeviceOptions(
     }
 
     /// <summary>
-    /// Deletes a device.
+    /// Deletes devices.
     /// </summary>
-    /// <param name="id">Device Id.</param>
+    /// <param name="ids">Device Ids.</param>
     /// <response code="204">Device deleted.</response>
     /// <response code="404">Device not found.</response>
-    /// <returns>A <see cref="NoContentResult"/> on success, or a <see cref="NotFoundResult"/> if the device could not be found.</returns>
+    /// <returns>A <see cref="NoContentResult"/> on success, or a <see cref="NotFoundResult"/> if a device could not be found.</returns>
     [HttpDelete]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult> DeleteDevice([FromQuery, Required] string id)
+    public async Task<ActionResult> DeleteDevice([FromQuery] string[] ids)
```

Comment:
Done in [`8e33f2a` (#12834)](https://github.com/jellyfin/jellyfin/pull/12834/commits/8e33f2ad1d948e0096db447c88aa2da5898bd610)

Reply:
Since this is backwards compatible, should changes to the web be _necessary_ for this then?

Reply:
No that should no longer be necessary as the existing requests will just work. Although it would be nice to eventually migrate that delete all button for performance.